portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/usecase/UIInfrastructureUsecase.testcase

```diff
@@ -20,6 +20,43 @@ definition {
 		}
 	}
 
+	@priority = "5"
+	test CheckAUIPreloadConfiguration {
+		property test.name.skip.portal.instance = "UIInfrastructureUsecase#CheckAUIPreloadConfiguration";
+
+		ProductMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Control Panel",
+			portlet = "System Settings");
+
+		SystemSettings.gotoConfiguration(
+			configurationCategory = "Third Party",
+			configurationName = "AUI",
+			configurationScope = "System Scope");
+
+		FormFields.viewCheckboxChecked(fieldName = "enableAUIPreload");
+
+		SystemSettings.configureSystemSetting(
+			enableSetting = "false",
+			settingFieldName = "Preload Widely Used AUI Modules");
+
```

Comment:
Maybe we can add a `FormFields.viewCheckboxNotChecked(fieldName = "enableAUIPreload");`

for https://issues.liferay.com/browse/LPS-112311?focusedCommentId=2160988&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2160988